source/integrations/zoom.rst

```diff
@@ -0,0 +1,61 @@
+Zoom Plugin
+================================
+
+With `Zoom integration <https://github.com/mattermost/mattermost-plugin-zoom>`_, Mattermost users can launch and join voice and video meetings without leaving 
+the Mattermost interface. In addition to being able to customize Zoom, users can also extend the plugin 
+to other video conferencing systems such as Skype and Google Hangouts.
+
+`Zoom <https://zoom.us/>`_ is one of the world’s leading video and web conferencing services. This 
+documentation includes instructions on how you can set up Zoom in Mattermost.
+
+Zoom Setup Guide
+~~~~~~~~~~~~~~~~~
+
+1. The System Console settings live in **System Console > Plugins > Zoom**. This is where the admin can
+configure Zoom for Mattermost.
+
+.. image:: ../images/zoom_system_console.png
+
+2. There are steps where you need to change a few settings in Zoom - see below instructions to enable API Key,
+API Secret, and Webhook Secret.
+
+API Key
+.........................................
+
+This is the API Key generated by Zoom, which is used to create meetings and pull user data. To generate an API Key:
+
+1. Go to https://developer.zoom.us/me/ and login.
+2. Switch to the API tab and click **Enable**, if not already enabled.
+3. Switch back to the API tab and copy the API Key.
+4. Paste the API Key into the field in the system console.
+
+API Secret
+.........................................
+
+This is the API Secret generated by Zoom, which is used to create meetings and pull used data. To generate an API Secret:
+
+1. Go to https://developer/zoom.us/me/ and login.
+2. Switch to the API tab and click **Enable**, if not already enabled.
+3. Switch back to the API tab and copy the API Secret.
+4. Paste the API Secret into the field in the system console.
+
+.. image:: ../images/zoom_api_key.png
+
+Webhook Secret
+.........................................
+
+This is the Secret used to authenticate the webhook to Mattermost. Use the webhook URL to set up the 
+Zoom webhook. Then hit **Save**.
+
+.. image:: ../images/zoom_webhook.png
+
+3. The setting for overriding usernames and icon URLs need to be set to true.
+
+4. Once these steps are done, activate the plugin at **System Console > Plugins > Management**.
+
+.. image:: ../images/zoom_system-console_management.png
+
+5. Once activated, you will see a video icon in the channel header. There are three different actions 
```

Comment:
How does one access the three different actions from one button, is it a drop down? I would propose rewording as it's a bit confusing.